<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MangaDownloadDialog</class>
 <widget class="QDialog" name="MangaDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Manga:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mangaNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Manga Link:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mangaLinkLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="downloadAllChaptersButton">
       <property name="text">
        <string>Download All Chapters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadLatestChapterButton">
       <property name="text">
        <string>Download Latest Chapter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkForNextIssueButton">
       <property name="text">
        <string>Just Check For Next Published Issue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
